invenio_administration/assets/semantic-ui/js/invenio_administration/search/SearchResultItem.js

```diff
@@ -23,12 +43,40 @@ export class SearchResultItem extends Component {
             </Table.Cell>
           );
         })}
+        {resourceHasActions && (
+          <Table.Cell>
+            <Actions
```

Comment:
Should this not to be a modal? Right now the action will trigger immediately on the button-click, no?

Reply:
I opened an issue to add a modal, I mentioned this on the standup